src/pybind/mgr/rook/module.py

```diff
@@ -134,6 +134,20 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
+class RookEnv(object):
+    def __init__(self):
+        pod_namespace = os.environ.get('POD_NAMESPACE', 'rook-ceph')
+        self.cluster_name = os.environ.get('ROOK_CLUSTER_NAME', pod_namespace)
+        self.cluster_ns = os.environ.get('ROOK_CLUSTER_NS', self.cluster_name)
```

Comment:
todo: remove ROOK_CLUSTER_NS

Reply:
I believe `self.cluster_ns` should be set to `pod_namespace`. The namespace could be different from the cluster CR name and `cluster_ns` represents the namespace where the mgr is running, right?

Reply:
fixed

Reply:
fixed